src/ekobox/member.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
```

Comment:
Wait: this looks a lot like https://github.com/N3PDF/eko/blob/29f0cae94d3f67e60d29321e01bc4c0f22c675f8/src/eko/member.py - why do we need a copy here? - Can someone remind me please what we had in mind with `import ekos product from @giacomomagni scripts`

Reply:
Consistent `eko`s can be composed, simply by matrix multiplication, and I guess @giacomomagni did it for the backward study. We wanted to keep the work done, and `ekobox` should be a suitable place.

Reply:
So the product between ekos is done for example [here](https://github.com/N3PDF/eko/blob/497773932f048a8e0c5873be87df082dd4dd0b25/backward_paper/runner.py#L92). 
There should be something similar also in the eko paper repo. 
But why do you need to repeat `meber.py`?

Reply:
This I don't know, no idea.